composer.json

```diff
@@ -19,6 +19,7 @@
         "magento/module-theme": "100.2.*||101.0.*",
         "magento/module-ui": "101.0.*||101.1.*",
         "ext-json": "*",
+        "ext-simplexml": "*",
```

Comment:
Is this needed as a result of the code added in this PR?

Reply:
Yes, the payload from Kount is in XML format, so I'm making use of SimpleXml. By default, it's enabled in PHP, but I think it's safe to include it as an additional safety net